ros2_foxglove_bridge/src/message_definition_cache.cpp

```diff
@@ -137,7 +164,16 @@ const MessageSpec& MessageDefinitionCache::load_message_spec(
                                 definition_identifier.package_resource_name);
   }
   const std::string package = match[1].str();
-  const std::string filename = match[2].str() + extension_for_format(definition_identifier.format);
+  const std::string subfolder = match[2].str();
+  const std::string type_name = match[3].str();
+  const bool is_action_type = subfolder == "action";
+
+  // The action type name includes the subtype which we have to remove to get the action name.
+  // Type name: Fibonacci_FeedbackMessage -> Action name: Fibonacci
+  const std::string action_name = type_name.substr(0UL, type_name.find('_'));
```

Comment:
do we need to handle type names that also contain underscores? do we need to verify that the suffix we stripped off is actually one of the suffixes we support?

Reply:
Yes, we should. Will do that